src/pydvl/value/shapley/classwise.py

```diff
@@ -0,0 +1,301 @@
+"""
+Refs:
+
+[1] CS-Shapley: Class-wise Shapley Values for Data Valuation in Classification (https://arxiv.org/abs/2211.06800)
```

Comment:
We use bibtex for citations. You can see how it's done elsewhere with `:footcite:` . You will need to add the bibliographic data to the bib file in the docs